tardis/energy_input/energy_source.py

```diff
@@ -350,3 +253,11 @@ def read_artis_lines(isotope, path_to_data):
         sep="  ",
         index_col=False,
     )
+
+
+def get_nuclear_lines_database(
+    path="~/Downloads/tardisnuclear/decay_radiation.h5",
```

Comment:
does the decay radiation data base always live in `~/Downloads/tardisnuclear`? this seems dangerous

Reply:
It's a kwarg because it's the standard location from `nuclear`, but the user can override